From d0b94f660ac503f7fcc79a03134da2f0c2853eb6 Mon Sep 17 00:00:00 2001
From: Oliver Smith <osmith@sysmocom.de>
Date: Sun, 10 Apr 2022 14:16:05 +0200
Subject: [PATCH] usb: musb: direct musb debug output to kernel log

Megi recommended we apply this patch to make getting a trace easier, now
it's all in dmesg.
---
 drivers/usb/musb/musb_trace.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/musb/musb_trace.c b/drivers/usb/musb/musb_trace.c
index 476872adce80..6bbbaa137608 100644
--- a/drivers/usb/musb/musb_trace.c
+++ b/drivers/usb/musb/musb_trace.c
@@ -19,6 +19,7 @@ void musb_dbg(struct musb *musb, const char *fmt, ...)
 	vaf.fmt = fmt;
 	vaf.va = &args;
 
+	vprintk(fmt, args);
 	trace_musb_log(musb, &vaf);
 
 	va_end(args);
-- 
2.30.2

